app-cdr/uif2iso/uif2iso-0.1.7c-r1.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+DESCRIPTION="Convert CD images from uif (MagicISO) to iso"
+HOMEPAGE="http://aluigi.altervista.org/mytoolz.htm#${PN}"
```

Comment:
That's not really helpful advice. All the sensible things that use the HOMEPAGE metadata require a run through metadata generation anyway, some code uses things like `HOMEPAGE+=" "` to add a second home page from a default added by the eclass.

The majority of `dev-perl/` basically doesn't even define a homepage, and uses the eclass and variable substitution to compute it.

So basically, you're outta-luck if you're expecting to be able to rely on homepages without a metadata run.

Saying homepages shouldn't have variables because editors can't click them is about as useful as saying SRC_URI shouldn't have variables because you cant download them with `curl $( grep SRC_URI file | sed )`.

If that's a real problem, that's a "use the right tools, don't make portage worse for no reason"

And it also makes issues like https://github.com/gentoo/gentoo/pull/4718#issuecomment-303134168 a big waste of everyones time to solve what problem? 

Maybe there's sense in arguing "for this package, the benefit is too small and only creates problems later", but the extrapolation of using that advice generically for the whole tree without thinking about it? I must disagree.